### DISCUSSION TITLE
Meeting Agendas and Action Points

### DISCUSSION THREAD

**Discussion by mrchristian** [semanticClimate/ipcc-report-cataloguing#2] [General]
General Meeting; Mon Oct 03 07:30 2023 UTC
Present: PMR; SW
APs: 

- [x] Change Slack name
- [x] Move discussion to semanticClimate GitHub
- [ ] Clone main repo from PMR to sC GitHub - when clean version available - Main will stay on Peter
- [ ] Cataloguing in progress - [Spreadsheet](https://docs.google.com/spreadsheets/d/18LcilHbXDpP2dybpFxXHaplnFBNAguU3QssqmJXaUVk/edit#gid=202882633)

**Comment by mrchristian** [comment]
General Meeting; Tue Oct 03 07:30 2023 UTC
Present: PMR; SW; RK
APs: 

- [ ] Mapping reports for AR6 and finding HTML sources - See discussion #3

**Comment by mrchristian** [comment]
General Meeting; Wed Oct 04 07:30 2023 UTC
Present: PMR; SW; RK
APs: 

- [ ] Checklist for internship onboarding RK
- [ ] Document checklist on Wiki RK/SW
- [ ] We need a GitHub wiki for organisational documentation SW
- [ ] Documentation of AR6, WG3 as an example: See: [semanticClimate/tree/main/ipcc/ar6/test](https://github.com/petermr/semanticClimate/tree/main/ipcc/ar6/test) - PMR
- [ ] Gatsby buddy invite @danielJ

**Reply by flower1430** [reply]
Today three new interns Kaustubh, Poulomi and Jahnavi has joined the semantic climate.

**Comment by flower1430** [comment]
THURSDAY 5/10/2023
- [ ] Send the welcome email to all the new interns
- [ ] Shared the orientation document with interns
- [ ] New Wikibase being setup at TIB (provisional agreement): Reason for this is better RDF output and cement ties to NFDI/TIB service [Wikibase4Research](https://nfdi4culture.de/services/details/wikibase4research.html) - example [Docker instance](https://gitlab.com/nfdi4culture/ta1-data-enrichment/wikibase)
- [ ] TIB meeting being setup and [Open Research Knwoledge Graph](https://orkg.org/)
- [ ] Letters fo support from TIB and NFDIs being sought
- [ ] The introduction about the semantic climate tools (https://semanticclimate.org/p/en/docs/)
- [ ] Discussed about the software needs to be installed (python, pygetpapers and docanalysis)
- [ ] The selection of chapter of interest from the IPCC reports by the Interns

**Comment by mrchristian** [comment]
FRIDAY 6/10/2023

### Agenda

1. Do we have a top level shared Google drive? If so can I get access as I have documents to deposite - Google user mrchristian001@gmail.com
2. Documents to create as shared idea pad: Nov IPCC presentation ideas
3. Documents to create as shared idea pad: Use cases and benefits that can be gained from semantic indexing of IPCC reports
4. Can I create a clone of 'semantic-climate' repo on PMR GitHub onto semanticclimate GitHub org - so users ariving at sematicClimate can see the code and a path back to PMR source.
5. Accomodation - Hackthehackathon - Ive had no word since mailing on Sunday - SW

### Action points

- [ ]

**Reply by flower1430** [reply]
- This folder "semantic_climate-resources" ( https://drive.google.com/drive/folders/18-ioesK_1TVsPlA6oKBGQmVmUL_0S7Pp?usp=drive_link ) has been created to deposit the documents.
- Document has been created to share ideas, cases and benefits in the semantic_climate-resources folder

**Comment by flower1430** [comment]
Possible task for Intern_ Next week (9th October-13th October, 2023)
- Installation of all the software and tools by each intern
- selection of chapters from IPCC reports
- Checkout petermr/semanticClimate repo to give brief idea about the resources with examples from previous analysis
- Daily record of progress in the orientation document by each Intern

**Comment by flower1430** [comment]
FRIDAY meeting with PMR, PG and RK

- Poulomi will going to work on cities" WG2/Ch6
- The glossary and one of the term present in them was explained to us by PMR
- I have written few lines about our Barcamp session in the [document](https://docs.google.com/document/d/1inTZx5leYSdfufELijXXICqZ2V3EgcBdNDG3SGKmjls/edit#heading=h.mwavt9fyjuhn) shared to us by Guido

**Comment by flower1430** [comment]
MONDAY meeting with RK, PG, KP and GT
- Gayatri Joined today and introduced herself to the team.
- Orientation document shared with them to maintain attendance and daily progress
- KP has chose to work on WG1/Chapter9
- Discussion about the installation of the tools
- Description of the IPCC glossary to the intern 
- The input and output folder of the glossary was explained and how to see any differences in them by comparing the two.

**Comment by petermr** [comment]
Well done everyone . Looking forward to knowing what chapters you have
selected.

**Comment by flower1430** [comment]
TUESDAY 10/10/2023 meeting with RK, PG and GT

Chapters selected by the Interns:
- Gayatri Taneja selected WG2/Ch18
- KP has chose to work on WG1/Chapter9
- Poulomi is interested in WG2/Ch6
-
Gayatri has written code to compare the glossary to find the differences. She has initially applied to few words. Today she will try to modify the code for the input and output directory with all the words.

**Comment by flower1430** [comment]
THURSDAY 12/10/2023 meeting with PMR, RK, GT

- Discussion about the chapters selected
- Gayatri has explained the code for getting some information about the glossary terms
- Gayatri will compare the "G" letter word of the glossary 
- RK will see the "R" letter word
- For comparing the input and output : the following points needs to be considered
	- title
	- any extra symbol
	- parent term in the description
	- button etc.

**Comment by flower1430** [comment]
MONDAY 16/10/2023 meeting with PMR, RK, GT, PG
- discussion about the total glossary
- PMR explained how to make discussion in git hub and its usage ( https://github.com/petermr/semanticClimate/discussions/37 )
- description of character conversion in glossary (input and output)
- Explained git hub repository about the code and semantic climate 
- About to learn and use git clone and other git commands

**Comment by petermr** [comment]
Thanks,
You can also point to specific timepoints in the recordings. Should be in
#dictionaries

On Mon, Oct 16, 2023 at 10:32 AM flower1430 ***@***.***>
wrote:

> MONDAY 16/10/2023 meeting with PMR, RK, GT, PG
>
>    - discussion about the total glossary
>    - PMR explained how to make discussion in git hub and its usage (
>    petermr/semanticClimate#37
>    <https://github.com/petermr/semanticClimate/discussions/37> )
>    - description of character conversion in glossary (input and output)
>    - Explained git hub repository about the code and semantic climate
>    - About to learn and use git clone and other git commands
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/semanticClimate/discussions/2#discussioncomment-7290643>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAFTCS4IQ5M4VS5NJGWHZKDX7T5LHANCNFSM6AAAAAA5QUQVAA>
> .
> You are receiving this because you commented.Message ID:
> <semanticClimate/ipcc-report-cataloguing/repo-discussions/2/comments/7290643
> @github.com>
>


-- 
Peter Murray-Rust
Founder ContentMine.org
and
Reader Emeritus in Molecular Informatics
Dept. Of Chemistry, University of Cambridge, CB2 1EW, UK

**Comment by flower1430** [comment]
TUESDAY 17/10/2023 meeting with PMR, GY, RK, GT, PG and new interns
- Introduction of semantic climate and the team
- Brief description of IPCC reports and glossary

**Comment by flower1430** [comment]
THURSDAY 19/10/2023 meeting with PMR, RK, GT, R and TS
- Overview of the IPCC glossary
- Introduction of git commands and repository
